client/src/js/components/AllAmenities.jsx

```diff
@@ -1,60 +1,47 @@
 import React from 'react';
 import AmenitySectionListing from './AmenitySectionListing';
 
-class AllAmenities extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      showAll: false,
-    };
-  }
-
-  showAmenities() {
-    this.setState({ showAll: !this.state.showAll });
-  }
-
-  render() {
-    const allAmenitiesList = this.props.data.map((category, i) => {
-      if (category.contents.length === 0) {
-        return null;
-      }
-
-      if (i === this.props.data.length - 1) {
-        return (
-          <div key={i}>
-            <div className="amenityCategorySection">
-              <div className="title_11" >{category.title}</div>
-            </div>
-            <div className="amenityNotIncluded">
-              <div className="itemsSection">
-                <AmenitySectionListing data={category.contents} />
-              </div>
-            </div> 
-          </div>
-        );
-      }
+const AllAmenities = (props) => {
+  const allAmenitiesList = props.data.map((category, i) => {
+    if (category.contents.length === 0) {
+      return null;
+    }
 
+    if (i === props.data.length - 1) {
       return (
-        <div key={i}>
+        <div key={category.title}>
           <div className="amenityCategorySection">
             <div className="title_11" >{category.title}</div>
           </div>
-          <div className="itemsSection">
-            <AmenitySectionListing data={category.contents} />
+          <div className="amenityNotIncluded">
+            <div className="itemsSection">
+              <AmenitySectionListing data={category.contents} />
+            </div>
           </div>
         </div>
       );
-    });
+    }
 
     return (
-      <hgroup id="allAmenitiesBoard" >
-        <div className="title_10" >Amenities</div>
-        <div className="amenityTable" >
-          {allAmenitiesList}
+      <div key={category.title}>
+        <div className="amenityCategorySection">
+          <div className="title_11" >{category.title}</div>
         </div>
-      </hgroup>
+        <div className="itemsSection">
+          <AmenitySectionListing data={category.contents} />
```

Comment:
Consider using a different name than data as that name could get confused with the normal data attribute